h2o-algos/src/main/java/hex/tree/gbm/GBM.java

```diff
@@ -916,6 +917,19 @@ private void rollupMinMaxPreds(DTree tree, DTree.Node node, float[] mins, int mi
       return new GBMModel(modelKey, parms, new GBMModel.GBMOutput(GBM.this));
     }
 
+    @Override
+    protected void doInTrainingCheckpoint() {
+      try {
+        String modelFile = _parms._in_training_checkpoints_dir + "/" + _model._key.toString() + "." + _model._output._ntrees;
```

Comment:
I'd be more explicit in the naming, sth like `+".ntrees_"+_ntrees`.
`.5` for example looks too much like a versioning system (e.g. when you save override a model, you backup the existing version by adding a number).
Might also be useful for someone looking directly at the folder and wanting to re-import those models directly.

Reply:
Adjusted